src/Providers/WayfairRouteServiceProvider.php

```diff
@@ -103,6 +66,12 @@ public function map(Router $router) {
         'middleware' => ['oauth.cookie']
         ]
     );
+    $router->delete(
+        'wayfair/fullInventory', [
+        'uses'       => 'Wayfair\Controllers\FullInventoryController@clearState',
```

Comment:
Not sure how the `router` works, but is `FullInventoryController` supposed to have a method `clearState` in it?

Reply:
Yes, and it does. it inherits the method from `InventoryController`.